lighthouse-core/test/computed/metrics/timing-summary-test.js

```diff
@@ -7,59 +7,63 @@
 
 const TimingSummary = require('../../../computed/metrics/timing-summary.js');
 
-const trace = require('../../fixtures/traces/progressive-app-m60.json');
-const devtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
+const trace = require('../../fixtures/traces/lcp-all-frames-m89.json');
+const devtoolsLog = require('../../fixtures/traces/lcp-all-frames-m89.devtools.log.json');
 
 /* eslint-env jest */
 describe('Timing summary', () => {
   it('contains the correct data', async () => {
```

Comment:
I thought this test could just be refreshed with a new trace. Would it be better to just add another one?

Reply:
updating seems fine, the guts of this are actually tested by metrics-test.js :)